src/components/navbar/NavbarCollapse.tsx

```diff
@@ -0,0 +1,7 @@
+import { FC } from 'react';
+import { useNavbarContext } from './NavbarContext';
+
+export const NavbarCollapse: FC = ({ children }) => {
+  const { isOpen } = useNavbarContext();
+  return <div className={isOpen ? 'w-full md:block md:w-auto' : 'hidden w-full md:block md:w-auto'}>{children}</div>;
```

Comment:
We can use `classNames` library to just have a condition on the `hidden` class name since it is the only one that depends on the `isOpen` property.